content/zh-cn/case-studies/netease/index.html

```diff
@@ -1,105 +1,154 @@
 ---
 title: 案例研究：NetEase
+linkTitle: NetEase
 case_study_styles: true
 cid: caseStudies
-css: /css/style_case_studies.css
+logo: netease_featured_logo.png
+featured: false
+
+new_case_study_styles: true
+heading_background: /images/case-studies/netease/banner1.jpg
+heading_title_logo: /images/netease_logo.png
```

Comment:
Should this actually be the SVG image?

Reply:
I think not,I try to change it into .svg,but not found,I think it should be same with en language